src/workers/userUpdatedCio.ts

```diff
@@ -24,7 +24,12 @@ const worker: TypedWorker<'user-updated'> = {
       await resubscribeUser(cio, user.id);
     }
 
-    await identifyUser(log, cio, user);
+    await identifyUser({
+      con,
+      log,
```

Comment:
we can remove log, and just import logger. So we don't have to keep passing it around

Reply:
yeah, I used it in new function, will also add here while I am refactoring it either way

Reply:
Done in 9fe14dff7e5a4186290b734209f74fe4200368a5